src/pudl/transform/vcerare.py

```diff
@@ -310,14 +401,18 @@ def one_year_hourly_available_capacity_factor(
         .pipe(_drop_city_cols, df_name)
         .pipe(_make_cap_fac_frac, df_name)
         .pipe(_stack_cap_fac_df, df_name)
+        .pipe(_clip_unexpected_2016_pv_capacity, df_name, year)
         for df_name, df in raw_dict.items()
     }
+
     # Combine the data and perform a few last cleaning mechanisms
     # Sort the data by primary key columns to produce compact row groups
     return apply_pudl_dtypes(
         _combine_all_cap_fac_dfs(clean_dict)
+        .pipe(_handle_2015_nulls, year)
         .pipe(_combine_cap_fac_with_fips_df, fips_df)
-        .sort_values(by=["state", "county_or_lake_name", "datetime_utc"])
+        .pipe(_standardize_census_names, census_pep_data)
+        .sort_values(by=["state", "place_name", "datetime_utc"])
```

Comment:
I fiddled around with _standardize_census_names a bit locally, but couldn't get the memory usage down significantly. If we can do this string manipulation earlier on in the process before the county names get broadcast across tens of millions of rows that would probably be much faster and less memory intensive. While they are still column headers early on? Or failing that, even just moving it up to the previous stanza, where we're operating on one type of generation at a time, rather than after they've been concatenated?